content/en/blog/_posts/2021-08-30-volume-populators-alpha.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Volume Populators Redesign
+date: 2021-08-30
+---
+
+**Authors:**
+Ben Swartzlander (NetApp)
+
+Kubernetes v1.22 introduces a redesigned approach for volume populators. Originally implemented
+in v1.18, the API suffered from backwards compatibility issues and v1.22 includes a new API
+field called `dataSourceRef` that fixes these problems.
+
+## Data sources
+
+The PersistentVolumeClaim object got a new field called `dataSource` when snapshot and cloning
+support was added to Kubernetes. This field allowed new PVCs to point to other PVCs or
+VolumeSnapshots in the same namespace, and modified the normal provisioning process so that
```

Comment:
```suggestion
Earlier Kubernetes releases already added a `dataSource` field into the the
[PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) API, used for snapshots and for cloning quiescent volumes.
You could use the `dataSource` field when creating a new PVC, referencing either
an existing PVC or a VolumeSnapshot in the same namespace. That also
modified the normal provisioning process so that
```